packs/pathfinder-bestiary/brass-dragon-ancient-spellcaster.json

```diff
@@ -2397,7 +2397,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>The brass dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/gmr 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p>\n<ul>\n<li><strong>Flame</strong> The dragon breathes fire in a @Template[line|distance:100|traits:arcane,evocation,fire,damaging-effect]{100-foot line} that deals @Damage[16d6[fire]] damage (@Check[reflex|dc:39|basic] save).</li>\n<li><strong>Sleep Gas</strong> The dragon breathes a @Template[cone|distance:80|traits:arcane,enchantment,incapacitation,sleep]{80-foot cone} of sleep gas. Each creature in the area must succeed at a @Check[fortitude|dc:39] save or fall @UUID[Compendium.pf2e.conditionitems.Item.Unconscious] for [[/gmr 1d6 #Duration Sleep Gas]]{1d6 rounds}, or 1 minute on a critical failure.</li>\n</ul>"
+                    "value": "<p>The brass dragon breathes in one of two ways.</p>\n<p>The dragon can't use Breath Weapon again for [[/gmr 1d4 #Recharge Breath Weapon]]{1d4 rounds}.</p><ul><li><strong>Flame</strong> (arcane, fire) The dragon breathes fire in a @Template[line|distance:100] that deals @Damage[16d6[fire]] damage (@Check[reflex|dc:39|traits:arcane,fire|basic] save).</li><li><strong>Sleep Gas</strong> (arcane, incapacitation, sleep) The dragon breathes a @Template[cone|distance:80|traits:arcane,incapacitation,sleep] of sleep gas. Each creature in the area must succeed at a @Check[fortitude|dc:39|traits:arcane,incapacitation,sleep] save or fall @UUID[Compendium.pf2e.conditionitems.Item.Unconscious] for [[/gmr 1d6 #Duration Sleep Gas]]{1d6 rounds}, or 1 minute on a critical failure.</li></ul>"
```

Comment:
basic before traits. While it's unlikely to come up again we put parameters for inlines in the order presented in the style guide https://github.com/foundryvtt/pf2e/wiki/Style-Guide#inline-check-links

This is because with the old inline style we didn't have a set order and it made the script to swap from that to `@Check` a real bear.